napari/_qt/widgets/qt_welcome.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, parent) -> None:
         self._image.setObjectName('logo_silhouette')
         self._image.setMinimumSize(300, 300)
         self._version_label = QtVersionLabel(
-            trans._(f'version: {__version__}')
+            trans._('version: {version}').format(version=__version__)
```

Comment:
Why did this need reformatting @TimMonko?

Reply:
Translation/Localization?

Reply:
test_and_pull_requests.yml uses semgrep and it flagged an f-string inside trans._()

I dunno never heard of it before just was google searching 🙃

Reply:
Yeah the trans module can't handle f-strings.

Having said this, I would actually just do this here:

```suggestion
            f'napari {version}'
```

Reply:
I'm with @jni: this string is `napari` which doesn't have a translation as a proper noun and a version number.